content/hi/docs/reference/glossary/application-architect.md

```diff
@@ -0,0 +1,18 @@
+---
+title: एप्लीकेशन आर्किटेक्ट
+id: application-architect
+date: 2018-04-12
+full_link:
+short_description: >
+  किसी एप्लिकेशन के उच्च-स्तरीय डिज़ाइन के लिए जिम्मेदार व्यक्ति।
+
+aka:
+tags:
+  - user-type
+---
+
+किसी एप्लिकेशन के उच्च-स्तरीय डिज़ाइन के लिए जिम्मेदार व्यक्ति।
+
+<!--more-->
+
+एक आर्किटेक्ट यह सुनिश्चित करता है कि एक ऐप का कार्यान्वयन इसे अपने आसपास के घटकों के साथ एक स्केलेबल, रखरखाव योग्य तरीके से बातचीत करने की अनुमति देता है। आसपास के घटकों में डेटाबेस, लॉगिंग इन्फ्रास्ट्रक्चर और अन्य माइक्रोसर्विसेज शामिल हैं।
```

Comment:
कार्यान्वयन means execution. Don't we mean to say execution environment in this?